server/gateway.go

```diff
@@ -33,20 +33,19 @@ const (
 	defaultSolicitGatewaysDelay         = time.Second
 	defaultGatewayConnectDelay          = time.Second
 	defaultGatewayReconnectDelay        = time.Second
-	defaultGatewayRecentSubExpiration   = 5 * time.Second
+	defaultGatewayRecentSubExpiration   = 250 * time.Millisecond
 	defaultGatewayMaxRUnsubBeforeSwitch = 1000
 
 	oldGWReplyPrefix    = "$GR."
 	oldGWReplyPrefixLen = len(oldGWReplyPrefix)
 	oldGWReplyStart     = oldGWReplyPrefixLen + 5 // len of prefix above + len of hash (4) + "."
 
-	// The new prefix is "$GNR.<x>.<cluster>.<server>." where <x> is 1 character
-	// reserved for service imports, <cluster> is 8 characters hash of origin
-	// cluster name and <server> is 8 characters hash of origin server pub key.
-	gwReplyPrefix    = "$GNR."
+	// The new prefix is "_GR_.<cluster>.<server>." where <cluster> is 8 characters
```

Comment:
Since you changed from 8 to 6 maybe update comment.

Reply:
Good point.. Will do.